framework/wazuh/core/cluster/hap_helper/wazuh.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import re
+from collections import defaultdict
+from enum import Enum
+from typing import Callable, Optional
+
+from wazuh.agent import get_agents, reconnect_agents
+from wazuh.cluster import get_nodes_info
+from wazuh.core.cluster.control import get_system_nodes
+from wazuh.core.cluster.dapi.dapi import DistributedAPI
+from wazuh.core.cluster.utils import ClusterFilter
+
+
+class WazuhAPIMethod(Enum):
```

Comment:
This class is no longer used since we are directly using the DAPI. It should be removed.